cypress/integration/group2/mytools.ts

```diff
@@ -52,7 +52,9 @@ describe('Dockstore my tools', () => {
   describe('Should contain extended DockstoreTool properties', () => {
     it('visit another page then come back', () => {
       cy.get('a#home-nav-button').click();
-      cy.contains('Docker Tools and Workflows for the Sciences');
+      cy.contains('Twitter');
+      cy.contains('Requests');
+      cy.contains('You have no requests');
```

Comment:
Nitpick: No requests is the normal desired state, we probably don't need to mention that they have no requests

Reply:
Do you mean in the UI just hiding the requests section completely?